chromium_src/chrome/browser/ui/webui/about_ui.cc

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "third_party/re2/src/re2/re2.h"
+#define CHROME_INTERNAL_URLS_TO_BRAVE                                 \
+  const std::string CHROME = "Chrome";                                \
```

Comment:
I think some keywords could be more specific. How about making them more specific like below?
```
const std::string header = "Chrome URLs";
...
const std::string chrome_pages_header = "List of Chrome URLs";
...
const std::string chrome_internal_pages_header = "List of chrome://internals pages";
...
```